<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>71</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Maya PyQt Basic Dialog Demo</string>
  </property>
  <widget class="QLabel" name="desc_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>46</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_button">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>12</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Shape</string>
   </property>
  </widget>
  <widget class="QComboBox" name="shape_type_cb">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Sphere</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cube</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cylinder</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="name_le">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>13</y>
     <width>112</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>newShape</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>